### ISSUE TITLE
Adds invalid config in user's settings.json

### ISSUE THREAD

**Issue by TheAifam5** [rafamel/subtle-brackets#31]
Hey,
I spotted that this extension adds `editor.matchBrackets: false` always, which is invalid in VSCode 1.4x.x.

The `editor.matchBrackets` does not accept booleans anymore, but "always", "near", "never".

To disable that behaviour I used:
`"subtleBrackets.disableNative": false`

Regards,
TheAifam5

**Comment by evdokimovm** [comment]
@TheAifam5 we can fix it by ourself for the time being.

Just change in `~/.vscode/extensions/rafamel.subtle-brackets-3.0.0/out/extension.js` file:

https://github.com/rafamel/subtle-brackets/blob/716184c125d445e934159d139e68d5b83ab0bd4e/src/extension.ts#L15

to

```
.update('editor.matchBrackets', "never", true);
```

**Comment by neuromagus** [comment]
Nope. Still used old settings. Please, update this extension in Market. And  "added a commit to AbdelrahmanHafez/subtle-brackets that referenced this issue on Mar 25 " not added...

file: https://github.com/rafamel/subtle-brackets/blob/master/src/extension.ts

    .update('editor.matchBrackets', false, true);  

please, update correct.

**Comment by setanarut** [comment]
No version compatibility check in vscode extensions? I think this extension has been abandoned. Unbelievable.

**Comment by shellscape** [comment]
I've forked this to https://github.com/shellscape/vscode-atom-bracket-matcher and will be releasing an extension on open-vsx.org in the next few days which will include this fix

**Comment by neuromagus** [comment]
Well, I'm start using Emacs. But nice to start of fix this ~~crap~~ plugin.

**Comment by shellscape** [comment]
Quick update for everyone. I've completed my initial work to get the fork up to snuff and the extension is published:

https://marketplace.visualstudio.com/items?itemName=shellscape.shellscape-brackets
https://open-vsx.org/extension/shellscape/shellscape-brackets

The repo is here https://github.com/shellscape/vscode-shellscape-brackets and feel free to open issues there.
